projects/plugins/jetpack/extensions/extended-blocks/paid-blocks/with-upgrade-banner.jsx

```diff
@@ -71,13 +71,18 @@ export default createHigherOrderComponent(
 		}, [ setAttributes, hasParentBanner ] );
 
 		useEffect( () => {
+			const block = document.querySelector( `[data-block="${ clientId }"]` );
+			if ( ! block ) {
+				return;
+			}
+
 			let upgradeBannerContainer = document.querySelector(
 				`[data-block="${ clientId }"] > .jetpack-block-upgrade-banner-container`
```

Comment:
Not a blocker, but I guess we could do this now?

```js
let upgradeBannerContainer = block.querySelector( '.jetpack-block-upgrade-banner-container' );
```

Reply:
Nope, we still need the `>` to select the direct child (otherwise we might select the container of a nested paid block), but that only works if you specify the parent CSS selector (see https://stackoverflow.com/questions/3680876/using-queryselectorall-to-retrieve-direct-children).